EIPS/eip-6110.md

```diff
@@ -94,17 +90,15 @@ def parse_deposit_data(deposit_event_data) -> bytes[]:
 def little_endian_to_uint64(data: bytes) -> uint64:
     return uint64(int.from_bytes(data, 'little'))
 
-def event_data_to_deposit_request_rlp(deposit_event_data) -> bytes:
+def event_data_to_deposit_request(deposit_event_data) -> bytes:
     deposit_data = parse_deposit_data(deposit_event_data)
     pubkey = Bytes48(deposit_data[0])
     withdrawal_credentials = Bytes32(deposit_data[1])
-    amount = little_endian_to_uint64(deposit_data[2])
+    amount = deposit_data[2]   # 4 bytes uint64 LE
```

Comment:
Not making the endianness conversion here defers this responsibility to the Engine API handlers

Reply:
The idea is that this is already valid SSZ.

Reply:
Yeah, AFAIK the number is already converted to LE by the contract.